app/controllers/Mod.scala

```diff
@@ -108,6 +108,11 @@ final class Mod(
       }
     )
 
+  def kid(username: String) =
+    OAuthMod(_.ForceKid) { _ => me =>
```

Comment:
I think the `ForceKid` permission has an unfortunate name. Maybe `SetKidMode` instead?